test/validation/api/scheduler/scheduler.c

```diff
@@ -185,8 +185,19 @@ static void scheduler_test_wait_time(void)
 	upper_limit = odp_time_local_from_ns(5 * ODP_TIME_SEC_IN_NS +
 							ODP_WAIT_TOLERANCE);
 
-	CU_ASSERT(odp_time_cmp(diff, lower_limit) >= 0);
-	CU_ASSERT(odp_time_cmp(diff, upper_limit) <= 0);
+	if (odp_time_cmp(diff, lower_limit) <= 0) {
```

Comment:
Shouldn't this be < rather than <= ?

Reply:
It's c&p from timer code. And '<' vs '<=' is 1ns difference.